File: no_arma/python/process.py

if __name__ == "__main__":
    filename = "temp.txt"
    tmed_pisum = 0.023
    tmed_fib = 0.004
    f=open(filename,"w+")
    f.write("py\n")
    f.write("pisum," + str(tmed_pisum) + "\n")
    f.write("fib," + str(tmed_fib) + "\n")
    f.close()

File: with_arma/R/process.R
process <- function(){
    t <- c(0.04, 0.005)
    dfout <- data.frame(R=t)
    filename <- "temp.txt"
    row.names(dfout) <- c("pisum", "fib")
    cat("\ndf out:\n")
    print(dfout)

    # write to table
    write.table(file=filename, x=dfout, sep=",", row.names=T, col.names=T, quote=F)

    # read in 
    dfin <- read.table(filename, header=T, sep=",")
    cat("\n\ndf in: \n")
    print(dfin)
}

